### PR TITLE
docs: Add RFC example

### PR DESCRIPTION
**Pull request by bokket**
Signed-off-by: bokket <3100563328@qq.com>

### CODE REVIEW

**Comment by Xuanwo** [comment]
I think this RFC needs to be divided into two parts.

1. A RFC to [go-endpoint](https://github.com/beyondstorage/go-endpoint) to add TCP support
2. A RFC to hdfs to describe how will hdfs use `endpoint` (Maybe it's too clear that don't need a RFC)

**Comment by bokket** [comment]
> I think this RFC needs to be divided into two parts.
> 
> 1. A RFC to [go-endpoint](https://github.com/beyondstorage/go-endpoint) to add TCP support
> 2. A RFC to hdfs to describe how will hdfs use `endpoint` (Maybe it's too clear that don't need a RFC)

It can be transferred to beyondstorage/go-endpoint/pull?
If not , may be merge `0-example` and I go beyondstorage/go-endpoint again

**Comment by Xuanwo** [comment]
> It can be transferred to beyondstorage/go-endpoint/pull?

No, we cannot.

> If not , may be merge `0-example` and I go beyondstorage/go-endpoint again

LGTM, let's remove the rfc-11 and just keep the example.

File: docs/rfcs/0-example.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+- Possible Q&As
+
+## Compatibility
+
+<proposal's compatibility statement>
+
+## Implementation
+
+Explain what steps should be done to implement this proposal.
+
```
Comment (Xuanwo) [inline docs/rfcs/0-example.md]:
```suggestion
```

File: docs/rfcs/0-example.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
+- Author: (fill me in with `name <mail>`, e.g., bokket [bokkett@gmail.com](mailto:bokkett@gmail.com))
+- Start Date: (fill me in with today's date, YYYY-MM-DD)
+- RFC PR: [beyondstorage/go-service-hdfs#0](https://github.com/beyondstorage/go-service-hdfs/issues/0)
+- Tracking Issue: [beyondstorage/go-service-hdfs#0](https://github.com/beyondstorage/go-service-hdfs/issues/0)
+
+# RFC-0:
+
+- Updates: (delete this part if not applicable)
+  - [GSP-20](https://github.com/beyondstorage/go-service-hdfs/blob/master/docs/rfcs/20-abc): Deletes something
```
Comment (Xuanwo) [inline docs/rfcs/0-example.md]:
```suggestion
  - [RFC-20](https://github.com/beyondstorage/go-service-hdfs/blob/master/docs/rfcs/20-abc): Deletes something
```

File: docs/rfcs/0-example.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+- Updated By: (delete this part if not applicable)
+  - [GSP-10](https://github.com/beyondstorage/go-service-hdfs/blob/master/docs/rfcs/10-do-be-do-be-do): Adds something
```
Comment (Xuanwo) [inline docs/rfcs/0-example.md]:
```suggestion
  - [RFC-10](https://github.com/beyondstorage/go-service-hdfs/blob/master/docs/rfcs/10-do-be-do-be-do): Adds something
```

File: docs/rfcs/0-example.md [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+  - [GSP-1000](https://github.com/beyondstorage/go-service-hdfs/blob/master/docs/rfcs/1000-lalala): Deprecates this RFC
```
Comment (Xuanwo) [inline docs/rfcs/0-example.md]:
```suggestion
  - [RFC-1000](https://github.com/beyondstorage/go-service-hdfs/blob/master/docs/rfcs/1000-lalala): Deprecates this RFC
```

**Review by Xuanwo — APPROVED** [review]

**Pull request merged**